onsset/onsset.py

```diff
@@ -574,76 +596,61 @@ def grid_penalties(self):
 
         logging.info('Classify road dist')
         #define bins as -inf to 5, 5 to 10, 10 to 25, 25 to 50, 50 to inf
-        road_distance_bins = [float("-inf"),5,10,25,50,float("inf")]
+        road_distance_bins = [0,5,10,25,50,float("inf")]
         #define classifiers
         road_distance_labels = [5,4,3,2,1]
 
-        self.df[SET_ROAD_DIST_CLASSIFIED] = pd.cut(self.df[SET_ROAD_DIST], road_distance_bins,
-                                            labels=road_distance_labels).astype(float)
+        ROAD_DIST_CLASSIFIED = pd.cut(data_frame[SET_ROAD_DIST], road_distance_bins,
```

Comment:
Create a separate function for ROAD_DIST_CLASSIFIED so that you can test it (see comments later on `test_road_dict.py`

```python
def classify_road_distance(road_distance, bins=[0,5,10,25,50,float("inf")], labels=[5,4,3,2,1]):
    """Classify the road distance according to bins and labels

    Arguments
    ---------
    road_distance : pandas.Series
    bins : list
    labels : list
    """
    return pd.cut(road_distance, bins, labels=labels, include_lowest=True).astype(float)
```